### PR TITLE
Add command for white noise toggle.

### PR DESCRIPTION
**Pull request by grassbl8d**
Add command for white noise toggle.

### CODE REVIEW

**Comment by kzhovn** [comment]
Thanks for the PR! Could you change the icon to something that I'm not using elsewhere, maybe https://feathericons.com/?query=headphones ? Otherwise, everything looks good.

**Comment by grassbl8d** [comment]
Ok I'll look for one. Thanks!

**Comment by grassbl8d** [comment]
Hi @kzhovn  I made some changes to use feather-headphones but I'm not entirely sure how this works since this is my first itme using it.

Btw, I modified timer so that it initializes white onoise player even if it starts as not enabled. 

Thanks.

File: src/timer.ts [hunk 1]
```diff
@@ -37,9 +37,8 @@ export class Timer {
 		this.pomosSinceStart = 0;
 		this.cyclesSinceLastAutoStop = 0;
 
-		if (this.settings.whiteNoise === true) {
```
Comment (kzhovn) [inline src/timer.ts:40]: If you're initializing the white noise player on load every time, you don't need this conditional.

Comment (grassbl8d) [reply src/timer.ts:40]: Actualy I removed it.

**Pull request closed without merging**